be/src/runtime/exec_env.h

```diff
@@ -158,6 +159,12 @@ class ExecEnv {
     PluginMgr* plugin_mgr() { return _plugin_mgr; }
     RuntimeFilterWorker* runtime_filter_worker() { return _runtime_filter_worker; }
     Status init_mem_tracker();
+    RuntimeFilterCache* runtime_filter_cache() { return _runtime_filter_cache; }
+
+    void add_rf_event(const TUniqueId& query_id, int filter_id, const std::string& network, std::string&& msg);
+    void add_rf_event(const PUniqueId& query_id, int filter_id, const std::string& network, std::string&& msg);
+    void add_rf_event(const TUniqueId& query_id, int filter_id, std::string&& msg);
```

Comment:
Could you be kind to provie less method? 

```
    void add_rf_event(const TUniqueId& query_id, int filter_id, const std::string& network, std::string&& msg);
    void add_rf_event(const PUniqueId& query_id, int filter_id, const std::string& network, std::string&& msg);
```

Reply:
this method must deal with query_id in different formats.

Reply:
I know, is there any more elegant way?  Like wrapping those parameters into a strucutre? Could that be more readable and writable?